charts/olm/cert-manager-0.0.1/kustomization.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - 0001_secret.yaml
+  - 0002_issuer.yaml
+  - 0003_certificates.yaml
+  - 0004_server.yaml
+  - 0005_client.yaml
```

These file contents are not correct for the files in templates
```suggestion
  - 0000_namespace.yaml 
  - 0001_operatorgroup.yaml
  - 0002_subscription.yaml
  - 0003_cr.yaml 
```